### PR TITLE
[full-ci] [tests-only] Added test for sending share invitation to deleted user

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
Added Scenarios:
- Scenario Outline: send share invitation to deleted user

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/7947

## How Has This Been Tested?

- Locally
- :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiGraph/createUser.feature [hunk 1]
```diff
@@ -81,7 +81,7 @@ Feature: create user
   Scenario: user can be created with the name of the deleted user
     Given user "Brian" has been created with default attributes and without skeleton files
     And the administrator has assigned the role "Admin" to user "Alice" using the Graph API
-    And the user "Alice" has deleted a user "Brian" using the Graph API
+    And the user "Alice" has deleted a user "Brian"
```
[the remark this answers is not in the document]
Comment (grgprarup) [reply tests/acceptance/features/apiGraph/createUser.feature:84]: We are not using such text in the `Given` steps.

File: tests/acceptance/features/apiSharingNg/shareInvitations.feature [hunk 2]
```diff
@@ -924,3 +924,66 @@ Feature: Send a sharing invitations
...
+                  "itemNotFound: not found"
+                ]
+              },
+              "innererror": {
+                "type": "object",
+                "required": [
+                  "date",
+                  "request-id"
+                ]
+              }
```
Comment (PrajwolAmatya) [inline tests/acceptance/features/apiSharingNg/shareInvitations.feature:973]: Do we need this as well?

**Review by SwikritiT — APPROVED** [review]
LGTM

File: tests/acceptance/features/apiGraph/createUser.feature (hunk 1 above)
[the remark this answers is not in the document]
Comment (S-Panta) [reply tests/acceptance/features/apiGraph/createUser.feature:84]: But there are many `Given` step where this phrase is used in the testcode

[the remark this answers is not in the document]
Comment (saw-jan) [reply tests/acceptance/features/apiGraph/createUser.feature:84]: we are slowly refactoring them when there's chance

**Pull request merged**